plenum/test/view_change_service/test_lag_by_checkpoint.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+
+from plenum.test.delayers import cDelay
+from plenum.test.helper import sdk_send_random_and_check, checkViewNoForNodes, assertExp, get_pp_seq_no
+from plenum.test.node_catchup.helper import ensure_all_nodes_have_same_data
+from plenum.test.node_request.helper import sdk_ensure_pool_functional
+from plenum.test.stasher import delay_rules_without_processing
+from plenum.test.test_node import ensureElectionsDone
+from plenum.test.view_change_service.helper import trigger_view_change
+from stp_core.loop.eventually import eventually
+
+CHK_FREQ = 2
+
+
+@pytest.fixture(scope="module")
+def tconf(tconf):
+    old_chk_freq = tconf.CHK_FREQ
+    tconf.CHK_FREQ = CHK_FREQ
+    tconf.LOG_SIZE = CHK_FREQ * 3
+    yield tconf
+    tconf.CHK_FREQ = old_chk_freq
+
+
+def test_lag_less_then_catchup(looper,
```

Comment:
As I understand this is Test 4 from the PoA of INDY-2223. Do we have Test 5?

Reply:
This is the test from the comment. It's about lagged node before view_change. The 4th test was parametrized and moved into separate file. 5th test will be in Renata's PR, she is fixing it now.